draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1013,20 +1013,38 @@ handshake is as follows:
 
 When a subflow fails to operate within the MP-DCCP requirements, it is 
 necessary to fall back to the safe operation. This may be either falling back 
-to regular DCCP, or removing a problematic subflow. The main reason for 
-subflow failing is loss of MP-DCCP options. 
+to regular DCCP, or removing a problematic subflow. The main reasons for 
+subflow failing include: loss of MP-DCCP options, failure to negotiate protocol 
+version, or modification of payload data. 
 
 At the start of the MP-DCCP connection, the handshake ensures exchange of MP-DCCP
 options and thus ensures that the path is fully MP-DCCP capable. If during the
 handshake procedure it appears that DCCP-Request or DCCP-Response or DCCP-Ack
 messages don’t have the MP-DCCP options, the MP-DCCP connection will not be 
-established and the handshake should fall back to regular DCCP. The same
-fallback should take place if the endpoints fail to agree on a protocol
-version to use during the Multipath Capable feature negotiation.
+established and the handshake should fall back to regular DCCP. 
+
+The same fallback should take place if the endpoints fail to agree on a protocol
+version to use during the Multipath Capable feature negotiation, which is described
+in section 3.1. The protocol version negotiation distinguishes between negotiation
+for the initial connection establishment, and addition of subsequent subflows. If
+protocol version negotiation is not successful during the initial connection establishment, MP-DCCP connection will fall back
+to regular DCCP. Any subflow addition to an existing MP connection MUST use the
+same version negotiated for the first flow. If a new subflow is not able to use the
+negotiated protocol version, that subflow will be closed. 
```

Comment:
There is no version negotiation for subsequent flows created with MP_JOIN. So from my view this text is not required.

Reply:
ok. sentence starting 'Any subflow addition...' removed.